WalletWasabi/Extensions/VerboseOutputInfo.cs

```diff
@@ -0,0 +1,15 @@
+namespace NBitcoin.RPC
+{
+	public class VerboseOutputInfo
+	{
+		public VerboseOutputInfo(Money value, Script scriptPubKey)
+		{
+			Value = value;
+			ScriptPubKey = scriptPubKey;
```

Comment:
Can you add an enum with the type here?

Reply:
I was thinking about it, but we anyway have the ability to determine script types.

Reply:
Done